Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,12 +1,27 @@
 ﻿using System;
+using System.IO;
 
 namespace BootCamp.Chapter
 {
     class Program
     {
         static void Main(string[] args)
         {
-
+            var textLogger = new Logging.LogToText();
+            textLogger.LogOpenProgram();
+            try
+            {
+                int b = 0;
+                int i = 10 / b;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'i'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5150047)